symfony.lock

```diff
@@ -1,4 +1,61 @@
 {
+    "babdev/pagerfanta-bundle": {
```

Comment:
Do not commit this change to `symfony.lock`. Full `symfony.lock` file is only commited in projects. Here, this file exists only with minimal data to prevent Symfony Flex from destroying the repo on first install, but still allowing fresh recipes to be installed when instantiating a project.